cli/command/service.go

```diff
@@ -10,6 +18,260 @@ func NewServiceCommand(b *CommandBase) *ServiceCommand {
 	return &ServiceCommand{b}
 }
 
-func (e *ServiceCommand) Command() cli.Command {
-	return cli.Command{}
+func (s *ServiceCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "service",
+		Usage: "manage Layer0 services",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new service",
+				Action:    s.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "loadbalancer",
+						Usage: "attach the service to the specified load balancer",
+					},
+				},
+			},
+			{
+				Name:      "delete",
+				Usage:     "delete a service",
+				Action:    s.delete,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:      "get",
+				Usage:     "describe a service",
+				Action:    s.read,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:   "list",
+				Usage:  "list all services",
+				Action: s.list,
+			},
+			{
+				Name:      "logs",
+				Usage:     "get the logs for a service",
+				Action:    s.logs,
+				ArgsUsage: "NAME",
+				Flags: []cli.Flag{
+					cli.IntFlag{
+						Name:  "tail",
+						Usage: "number of lines from the end to return (default: 0)",
+					},
+					cli.StringFlag{
+						Name:  "start",
+						Usage: "the start of the time range to fetch logs (format: YYYY-MM-DD HH:MM)",
+					},
+					cli.StringFlag{
+						Name:  "end",
+						Usage: "the end of the time range to fetch logs (format: YYYY-MM-DD HH:MM)",
+					},
+				},
+			},
+			{
+				Name:      "scale",
+				Usage:     "scale a service",
+				Action:    s.scale,
+				ArgsUsage: "NAME COUNT",
+			},
+			{
+				Name:      "update",
+				Usage:     "run a new dploy on a service",
+				Action:    s.update,
+				ArgsUsage: "NAME DEPLOY",
+			},
+		},
+	}
+}
+
+func (s *ServiceCommand) create(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "ENVIRONMENT", "NAME", "DEPLOY")
```

Comment:
If you make the changes above, `NAME` must become `SERVICE_NAME`

Reply:
✅